### ISSUE TITLE
train on own dataset

### ISSUE THREAD

**Issue by Shawn0Hsu** [dd604/refinedet.pytorch#4]
Hi, thank you for your work. I want to train my own data set on this network. There are only a few types and the data format is VOC format. May I ask what I need to modify, thank you!

**Comment by dd604** [comment]
You can refer to the training on VOC, where data arranges as follow:
VOCdevkit2007
|__ VOC2007
    |_ JPEGImages
    |_ Annotations
    |_ ImageSets
    |_ SegmentationClass

**Comment by Shawn0Hsu** [comment]
> You can refer to the training on VOC, where data arranges as follow:
> VOCdevkit2007
> |__ VOC2007
> |_ JPEGImages
> |_ Annotations
> |_ ImageSets
> |_ SegmentationClass

Okay, thank you. I have tried training on my own data set. There is only one category, but the training loss function will soon be nan. Do you know the reason? The training set is about 2000 images.

**Comment by dd604** [comment]
> > You can refer to the training on VOC, where data arranges as follow:
> > VOCdevkit2007
> > |__ VOC2007
> > |_ JPEGImages
> > |_ Annotations
> > |_ ImageSets
> > |_ SegmentationClass
> 
> Okay, thank you. I have tried training on my own data set. There is only one category, but the training loss function will soon be nan. Do you know the reason? The training set is about 2000 images.

You can use a smaller learning rate.

**Comment by Shawn0Hsu** [comment]
> > > You can refer to the training on VOC, where data arranges as follow:
> > > VOCdevkit2007
> > > |__ VOC2007
> > > |_ JPEGImages
> > > |_ Annotations
> > > |_ ImageSets
> > > |_ SegmentationClass
> > 
> > 
> > Okay, thank you. I have tried training on my own data set. There is only one category, but the training loss function will soon be nan. Do you know the reason? The training set is about 2000 images.
> 
> You can use a smaller learning rate.

Ok, thanks. I will try, thank you!

**Comment by dd604** [comment]
> > > > You can refer to the training on VOC, where data arranges as follow:
> > > > VOCdevkit2007
> > > > |__ VOC2007
> > > > |_ JPEGImages
> > > > |_ Annotations
> > > > |_ ImageSets
> > > > |_ SegmentationClass
> > > 
> > > 
> > > Okay, thank you. I have tried training on my own data set. There is only one category, but the training loss function will soon be nan. Do you know the reason? The training set is about 2000 images.
> > 
> > 
> > You can use a smaller learning rate.
> 
> Ok, thanks. I will try, thank you!

No problem. You can also consider using warmup.

**Comment by shangshanghuiliyi** [comment]
Where to modify the data to find the path, I don't want to follow the kind of data file storage form given by the author
